product/utilities/miscellaneous/index.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Miscellaneous
+layout: page
+---
+
+<p class="t-4">A page dedicated to other various utility classes.</p>
+
+<hr />
+
+<h2 class="m-bottom-4">Border Radius</h2>
+
+#### All corners are specified by default.
+
+<div class="p-4 border u-border-radius m-bottom-4">
+		<p class ="m-0">Border radius</p>
+</div>
+
+{% highlight html %}
+<div class="p-4 border u-border-radius">
+	<p>Border radius</p>
+</div>
+{% endhighlight %}
+
+
+<hr />
+
+<h2 class="m-bottom-4">Clearfix</h2>
+
+#### The clearfix utility is used on elements that need to self-clear their child elements. Use <code>.u-clearfix</code> or <code>.u-cf</code> to prevent floated elements from overflowing.
+
+
+<p class="Alert m-top-4">
+<strong>Please Note:</strong> Don't forget to attach a wrapper with the <code>.u-clearfix</code> class to floats and legacy grids.</p>
+
+<div class="bg-c-g200 border-radius p-2 m-bottom-4">
+	<div class="u-clearfix m-top-2 m-bottom-2">
+		<div class="bg-c-g100 border-radius p-4 l-float-left"><strong>.l-float-left</strong></div>
+		<div class="bg-c-g100 border-radius p-4 l-float-right"><strong>.l-float-right</strong></div>
+	</div>
+	<div class="u-clearfix m-bottom-2">
```

Comment:
Would leave off this example using the `g-1_3` and similar classes. This is for the legacy grid, which will likely be going away soon. I think the float example illustrates the clearfix functionality well enough :)

Reply:
Fixed!